connectors/src/connectors/slack/index.ts

```diff
@@ -282,9 +283,16 @@ export async function cleanupSlackConnector(
       `Attempting Slack app deactivation [cleanupSlackConnector]`
     );
 
-    const uninstallRes = await uninstallSlack(connector.connectionId);
-    if (uninstallRes.isErr()) {
-      return uninstallRes;
+    try {
+      const uninstallRes = await uninstallSlack(connector.connectionId);
+
+      if (uninstallRes.isErr() && !force) {
+        return uninstallRes;
+      }
+    } catch (e) {
+      if (!force) {
+        throw e;
```

Comment:
Might want to logger.error() there no?

Reply:
errors are already logged in `uninstallSlack` before throwing the error